Destructured parameter in subscribe callback uses incorrect TypeScript syntax

**To Reproduce**
Steps to reproduce the behavior:
1. Run tsc on a project that uses react-native-zip-archive
2. See errors:

```
node_modules/react-native-zip-archive/index.d.ts:14:52 - error TS7031: Binding element 'number' implicitly has an 'any' type.

14   export function subscribe(callback: ({ progress: number, filePath: string }) => void): NativeEventSubscription;
                                                      ~~~~~~

node_modules/react-native-zip-archive/index.d.ts:14:70 - error TS7031: Binding element 'string' implicitly has an 'any' type.

14   export function subscribe(callback: ({ progress: number, filePath: string }) => void): NativeEventSubscription;
                                                                        ~~~~~~
```

**Expected behavior**
No TypeScript errors

**Env (please complete the following information):**
Any

--- a/index.d.ts
+++ b/index.d.ts
@@ -11,5 +11,5 @@ declare module 'react-native-zip-archive' {
   export function unzip(source: string, target: string, charset?: string): Promise<string>;
   export function unzipWithPassword(assetPath: string, target: string, password: string): Promise<string>;
   export function unzipAssets(assetPath: string, target: string): Promise<string>;
-  export function subscribe(callback: ({ progress: number, filePath: string }) => void): NativeEventSubscription;
+  export function subscribe(callback: ({ progress, filePath }: { progress: number, filePath: string }) => void): NativeEventSubscription;
 }

